iroha-cli/interactive/impl/interactive_transaction_cli.cpp

```diff
@@ -482,10 +482,12 @@ namespace iroha_cli {
 
       GrpcResponseHandler response_handler;
 
-      response_handler.handle(
-          CliClient(address.value().first, address.value().second).sendTx(tx));
+      if(response_handler.handle(
+          CliClient(address.value().first, address.value().second).sendTx(tx)))
```

Comment:
clang-format should add braces for that `if`, that's strange if it didn't do that
it's [not really mandatory](https://google.github.io/styleguide/cppguide.html#Conditionals), but i think it's better add them [braces]